docs/source/index.rst

```diff
@@ -58,7 +58,7 @@ API Reference
    api/utils/avocado.utils.rst
    api/core/avocado.core.rst
    api/plugins/avocado.plugins.rst
-   api/optional-plugins/index
+   api/optional-plugins/index.rst
```

Comment:
Oh, here it is.  I guess this was intended to be in 1675c6ba9be81839bfb776deeeb50032ba774850 instead.